scripts/downloadDependencies.ps1

```diff
@@ -1,14 +1,19 @@
 [cmdletbinding()]
 param (
     [Parameter()]
-    $BicepNetUrl = 'https://github.com/PSBicep/BicepNet/releases/download/v2.0.4/BicepNet.PS.zip',
-    
-    [Parameter(DontShow)]
-    $BaseUri = 'https://api.github.com'
+    $bicepNetVersion
 )
 
 try {
-    if($PSScriptRoot) {
+
+    if (-not $bicepNetVersion) {
+        $bicepNetVersion = Get-Content -Path "$PSScriptRoot\..\.bicepNetVersion"
```

Comment:
This should use -Raw so if someone accidentally adds a newline to the text file, we still only get one string

Reply:
Fixed!